scripts/x-charts.exports.json

```diff
@@ -35,7 +35,17 @@
   { "name": "BarElementProps", "kind": "TypeAlias" },
   { "name": "BarElementSlotProps", "kind": "Interface" },
   { "name": "BarElementSlots", "kind": "Interface" },
+  { "name": "BarItem", "kind": "ImportSpecifier" },
```

Comment:
This is an interesting `kind` for these imports...
I imagine that changing them for immediate exports would cause them to be resolved properly. 🤔

Reply:
Worked, thanks for the input